src/common/transformers/getOtherLine.js

```diff
@@ -1,3 +1,8 @@
 export default (context, targetColumn) =>
-    async prev =>
-        context.fetchLineBy(targetColumn, prev);
+    (prev) => {
```

Comment:
fetchLineBy return a promise aint it ?
In this case make this function async.
This way it will always return a promise, and it will be clear that it is async.